CHANGELOG.md

```diff
@@ -85,13 +85,32 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 - Added: Biotech Research Area 2 can now be crossed with a Hypermode Wall Boost
 
+##### Impact Crater
+
+##### Magmoor Caverns
+
+- Added: The Sloped R-Jump in Geothermal Core to get to the door to Plasma Processing (commonly referred to as Eagle Jump) is now in logic.
+- Changed: Geothermal Core: Various Puddle Spore requirements now require Before Storage Depot B instead of After.
+
+##### Phazon Mines
+
+- Changed: Central Dynamo: Infinite Speed trick only has one connection and no longer requires Knowledge (Advanced)
```

Comment:
Seeing this made me wonder: do we have a warning if someone tries to use IS in logic for MW games?

Reply:
Might be worth putting in an FAQ.

Reply:
I'd rather stop people from genning impossible seeds than them finding out during a playthrough

Reply:
It's the job of the database maintainers to reject any PRs which add IS trick logic which can result in colleting multiple items at once. It's impossible to grab another item on the same frame as Central Dynamo so we're good so far